1/contrib/s2i/run

```diff
@@ -9,6 +9,29 @@
 source /usr/local/bin/jenkins-common.sh
 source /usr/local/bin/kube-slave-common.sh
 
+#display the available i386 and x86_64 java's to confirm their presence and version
+alternatives --display java | grep family | grep i386
+alternatives --display java | grep family | grep x86_64
+
+#get the fully qualified paths to both 32 and 64 bit java
+JVMPath32bit=`alternatives --display java | grep family | grep i386 | awk '{print $1}'`
+JVMPath64bit=`alternatives --display java | grep family | grep x86_64 | awk '{print $1}'`
+
+# set the java version used based on OPENSHIFT_JENKINS_JVM_ARCH
+if [ -z $OPENSHIFT_JENKINS_JVM_ARCH  ]; then
+    echo "Using 64 bit Java since OPENSHIFT_JENKINS_JVM_ARCH is not set"
+    alternatives --set java $JVMPath64bit
+elif [ "${OPENSHIFT_JENKINS_JVM_ARCH}" == "x86_64"  ]; then
+    echo "64 bit Java explicitly set in OPENSHIFT_JENKINS_JVM_ARCH"
+    alternatives --set java $JVMPath64bit
+else
+    echo "OPENSHIFT_JENKINS_JVM_ARCH is set to something other than x86_64 so using 32 bit Java"
+    alternatives --set java $JVMPath32bit
+fi
+
+#confirm what version of java is being used
+ls -la /etc/alternatives/java
```

Comment:
Basic / same response as above  .... could see getting rid of the ones above and just keeping this one for that debug note on which JDK the customer went with